### PR TITLE
refactor(addslicecontainer): move slice container panel from bootstrap panel to ant-d

### PR DESCRIPTION
**Pull request by pkdotson**
### SUMMARY
^
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
before
Screen Shot 2021-04-08 at 2 57 08 PM

after
Screen Shot 2021-04-08 at 2 54 02 PM

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by junlincc** [comment]
@pkdotson thanks Phillip! do you know where else this component is using other than + Chart?

File: superset-frontend/src/addSlice/AddSliceContainer.tsx [hunk 1]
```diff
@@ -17,10 +17,11 @@
  * under the License.
  */
 import React from 'react';
-import { Panel } from 'react-bootstrap';
+// import { Panel } from 'react-bootstrap';
```
Comment (michael-s-molina) [inline superset-frontend/src/addSlice/AddSliceContainer.tsx:20]:
```suggestion
```

File: superset-frontend/src/addSlice/AddSliceContainer.tsx [hunk 2]
```diff
@@ -41,6 +40,14 @@ export type AddSliceContainerState = {
 };
 
 const styleSelectContainer = { width: 600, marginBottom: '10px' };
+const StyledContainer = styled.div`
+  border-radius: ${({ theme }) => theme.gridUnit}px;
+  background-color: white;
```
Comment (rusackas) [inline superset-frontend/src/addSlice/AddSliceContainer.tsx:45]: Let's use the theme color version of white... I think it's grayscale.light4 or something

File: superset-frontend/src/addSlice/AddSliceContainer.tsx [hunk 3]
```diff
@@ -88,69 +95,63 @@ export default class AddSliceContainer extends React.PureComponent<
...
-                labelType="primary"
-              />
-            </div>
-            <br />
-            <hr />
-            <Button
-              buttonStyle="primary"
-              disabled={this.isBtnDisabled()}
-              onClick={this.gotoSlice}
+      <StyledContainer className="container">
```
Comment (rusackas) [inline superset-frontend/src/addSlice/AddSliceContainer.tsx:98]: I'm not sure if this className does anything... you might be able to get rid of it.

File: superset-frontend/src/addSlice/AddSliceContainer.tsx (hunk 2 above)
Comment (rusackas) [reply superset-frontend/src/addSlice/AddSliceContainer.tsx:45]: ...light5. Off by 1 error ;)

**Review by rusackas — APPROVED** [review]
LGTM

**Comment by pkdotson** [comment]
> @pkdotson thanks Phillip! do you know where else this component is using other than + Chart?

This is it I believe.

**Pull request merged**